test/cmd/README.md

```diff
@@ -0,0 +1,44 @@
+# Kubernetes Command-Line Integration Test Suite
+
+This document describes how Kubernetes should interact with the Kubernetes command-line integration test suite.
```

Comment:
I don't think this reads right. I think you mean something like "This document outlines how you can use the command-line integration test-suite"